api/integrations/google/oauth.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import argparse
+import datetime
+import httplib2
+import os
+
+# Uses Google's Client Library
+#   pip install --upgrade google-api-python-client
+from oauth2client import client, tools
+from oauth2client.file import Storage
+
+# Before running this, make sure to register a new
+# project in the Google Developers Console
+# (https://console.developers.google.com/start)
+# and download the "client secret" JSON file.
+#
+# You'll have to specify its location with --secret-path.
+
+# This parent argparser is used because it already contains
+# the arguments that Google's Client Library method "tools.run_flow"
+# supports.
```

Comment:
Should've expected it from the `.readonly` that using the above scopes gives traceback:
```
INFO:googleapiclient.discovery:URL being requested: POST https://www.googleapis.com/calendar/v3/calendars/primary/events?alt=json
WARNING:googleapiclient.http:Encountered 403 Forbidden with reason "insufficientPermissions"
ERROR:root:<HttpError 403 when requesting https://www.googleapis.com/calendar/v3/calendars/primary/events?alt=json returned "Insufficient Permission">
Traceback (most recent call last):
  File "/Users/alicja/Google Drive/Zulip/zulip/contrib_bots/lib/gcalendar.py", line 269, in handle_message
    event = self.service.events().insert(calendarId='primary', body=event).execute()
  File "/usr/local/lib/python2.7/site-packages/oauth2client/_helpers.py", line 133, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/googleapiclient/http.py", line 838, in execute
    raise HttpError(resp, content, uri=self.uri)
HttpError: <HttpError 403 when requesting https://www.googleapis.com/calendar/v3/calendars/primary/events?alt=json returned "Insufficient Permission">
INFO:root:waiting for next message
```

Wondering how it was working earlier.